wiki/PlaceholderExpansion.md

```diff
@@ -58,19 +58,27 @@ public class SomeExpansion extends PlaceholderExpansion {
 Let's quickly break down the different methods you had to import.
 
 - #### getAuthor
-  Through this method can you set the name of who created the expansion.
+  Through this method you can set the name of who created the expansion.
 - #### getIdentifier
   The name that should be used to identify the placeholders for this expansion.  
   The Identifier is the first text after the `%` and before the first `_` (`%identifier_values%`) and can therefore not contain any `_` in it.
   
   If you want to use `_` in your Expansion's name can you override the optional `getName()` method.
 - #### getVersion
-  This is a String, which means it doesn't has to be a number in itself. The String is used to determine if a new update is available or not, when the expansion is shared on the eCloud.  
-  For Expansions that are part of a plugin does this not really matter.
+  This is a String, which means it doesn't have to be a number in itself. The String is used to determine if a new update is available or not, when the expansion is shared on the eCloud.  
```

Comment:
```suggestion
This is a string, which means it can contain more than just a number. This is used to determine if a new update is available or not when the expansion is shared on the eCloud.
```